src/css/rent-form.css

```diff
@@ -1,4 +1,163 @@
-.sec-rent-form {
-  background-color: black;
-  color: wheat;
+.rent-form {
+  padding: 80px 0;
+  background-color: #242424;
 }
+.rent-container{
+  max-width: 320px;
+  padding-left: 20px;
+  padding-right: 20px;
+  margin-left: auto;
+  margin-right: auto;
+}
+.rent-title{
+  font-weight: 500;
+font-size: 28px;
+line-height: 1;
+letter-spacing: -0.02em;
+color: #fbfbfb;
+}
+.rent-formular{
+  display: flex;
+align-items: flex-start;
+justify-content: flex-start;
+flex-direction: column;
+gap: 18px;
+min-width: 320px;
+max-width: 375px;
```

Comment:
320px and 375px it's a width of the screen, but not of the container. max-width must equal to 335px, like on the design (it's a width for 375px screens) and min-width you have to calculate, maybe 280px.